src/analog/commands/downlink/GetParameter.ts

```diff
@@ -1,33 +1,35 @@
 import Command, {TCommandExampleList} from '../../Command.js';
-import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import CommandBinaryBuffer, {IRequestParameter} from '../../CommandBinaryBuffer.js';
 import {DOWNLINK} from '../../constants/directions.js';
 import * as deviceParameters from '../../constants/deviceParameters.js';
 
 
-/**
- * GetParameter command parameters
- */
-interface IGetParameterParameters {
-    /**
-     * Parameter id - one of `constants/deviceParameters`.
-     */
-    id: number
-}
-
-
 const COMMAND_ID = 0x04;
-const COMMAND_BODY_SIZE = 1;
 
 const examples: TCommandExampleList = [
     {
         name: 'request absolute data (not multichannel device)',
-        parameters: {id: deviceParameters.ABSOLUTE_DATA},
+        parameters: {
+            id: deviceParameters.ABSOLUTE_DATA,
+            data: undefined
```

Comment:
```suggestion
            data: null
```

?

this field may be not visible after serialization in jooby-gui

Reply:
null added as possible value and by default